### ISSUE TITLE
Dorothy - read on link does not work from fresh clone

### ISSUE THREAD

**Issue by epetre** [cloudhead/toto#119]
Trying to see if this solution could work for us, so I setup

git clone git://github.com/cloudhead/dorothy.git myblog on heroku.

The problem is all read on links do not work, is there something I need to configure before it works?

I also followe this article (http://www.rubyinside.com/deploy-blog-with-toto-and-heroku-2962.html) and it does not show the full article when I click on any of them.

All I get is a 404

toto, we're not in Kansas anymore (404)

**Comment by epetre** [comment]
Aparently the problem is that the title of the article must be exactly the same as the file name.

**Comment by iammichiel** [comment]
Yes, indeed, You can also use the `:slug` parameter in your articles. But they still have too be identical to the file name. 

```
:slug: my-beautifull-title
```

Your filename will have to be `my-beautifull-title.txt`

Checkout this [article](http://fadeyev.net/2010/05/10/getting-started-with-toto/) for more information.

BTW this is not an issue, you should close it. ;)
